code/datums/spells/devil.dm

```diff
@@ -0,0 +1,211 @@
+/obj/effect/proc_holder/spell/targeted/summon_pitchfork
+	name = "Summon Pitchfork"
+	desc = "A devil's weapon of choice.  Use this to summon/unsummon your pitchfork."
+	invocation_type = "none"
+	include_user = 1
+	range = -1
+	clothes_req = 0
+	var/obj/item/pitchfork
+	var/pitchfork_type = /obj/item/weapon/twohanded/pitchfork/demonic/
+
+	school = "conjuration"
+	charge_max = 150
+	cooldown_min = 10
+	action_icon_state = "pitchfork"
+	action_background_icon_state = "bg_demon"
+
+/obj/effect/proc_holder/spell/targeted/summon_pitchfork/cast(list/targets, mob/user = usr)
+	if (pitchfork)
+		qdel(pitchfork)
```

Comment:
Might want to add a `pitchfork = null` line after this to guarantee instant re-use